packages/astro/src/cli/index.ts

```diff
@@ -69,62 +67,11 @@ function printAstroHelp() {
 }
 
 /** Display --version flag */
-async function printVersion() {
+function printVersion() {
 	console.log();
 	console.log(`  ${colors.bgGreen(colors.black(` astro `))} ${colors.green(`v${ASTRO_VERSION}`)}`);
 }
 
-async function printInfo({
```

Comment:
This chunk of code is moved to `src/info/index.ts`, replacing dynamic imports as normal imports, as the entire module would be dynamically imported instead.